cluster/openstack-heat/config-default.sh

```diff
@@ -35,6 +35,8 @@ MINION_FLAVOR=${MINION_FLAVOR:-m1.medium}
 
 EXTERNAL_NETWORK=${EXTERNAL_NETWORK:-public}
 
+LBAAS_VERSION=${LBAAS_VERSION:-v1}
```

Comment:
I agree with @anguslees, we can default to "" and let the plugin auto-detect which loadbalancer to use.